task_app/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from .models import Category, Task, SubTask
+
+@admin.register(Category)
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ('name',)
+    search_fields = ('name',)
+
+class SubTaskInline(admin.TabularInline):
+    model = SubTask
+    extra = 1
+    min_num = 0
+    can_delete = True
+
+@admin.action(description="Mark as 'Done'")
+def mark_as_done(self, request, queryset):
+    queryset.update(status="done")
+
+@admin.register(SubTask)
+class SubTaskAdmin(admin.ModelAdmin):
+    list_display = ('title', 'task', 'status', 'deadline')
+    list_filter = ('status',)
+    search_fields = ('title', 'description')
+    actions = [mark_as_done]
```

Comment:
Не обязательно это действие делать глобальным. Оно ведь связано только с конкретным классом
```suggestion
@admin.register(SubTask)
class SubTaskAdmin(admin.ModelAdmin):
    list_display = ('title', 'task', 'status', 'deadline')
    list_filter = ('status',)
    search_fields = ('title', 'description')
    
    @admin.action(description="Mark as 'Done'")
    def mark_as_done(self, request, queryset):
        queryset.update(status="done")

    actions = ['mark_as_done']
```